From fbd1a7331daa1d2a1a08c7af83f7a7989308d859 Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <shreenidhi.shedi@broadcom.com>
Date: Tue, 10 Sep 2024 17:02:42 +0530
Subject: [PATCH 1/8] This patch fixes a warning that is shown upon every rpm
 usage.

Without this change, package builder throws a warning message upon
building every package.

Signed-off-by: Shreenidhi Shedi <shreenidhi.shedi@broadcom.com>
---
 lib/backend/dbi.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/backend/dbi.cc b/lib/backend/dbi.cc
index e91a9e6..8947965 100644
--- a/lib/backend/dbi.cc
+++ b/lib/backend/dbi.cc
@@ -114,7 +114,7 @@ dbDetectBackend(rpmdb rdb)
 			    _("Converting database from %s to %s backend\n"),
 			    ondisk->name, db_backend);
 		} else {
-		    rpmlog(RPMLOG_WARNING,
+		    rpmlog(RPMLOG_DEBUG,
 			_("Found %s %s database while attempting %s backend: "
 			"using %s backend.\n"),
 			ondisk->name, ondisk->path, db_backend, ondisk->name);
-- 
2.43.7

